docs/tutorials/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _tutorials:
+
+=================
+Tutorials
+=================
+The following contain tutorials for how to utilize regolith commands that require editing the regolith.rc file
+
+.. toctree::
+    :maxdepth: 1
+
+    broker
+    mongo_maintenance
```

Comment:
I think this is the wrong place for this.  We want to separate "user-docs", "maintainer-docs" (a new category) and developer docs.    Developer docs don't appear here at all (docstrings etc. in the code).    I think maintainer docs could go here (why not, it is quite archival) but they should be well separated so we don't freak out users who wouldn't have to know about this stuff anyway.   Maybe we want to put "Maintaining production Regolith installations" or something somewhere way at the bottom and then put this tutorial down there?

Reply:
Addressed